variables.tf

```diff
@@ -3,6 +3,13 @@
 #
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
+variable "client_dns_search_domain" {
+  description = "The DNS search domain to be pushed to VPN clients."
```

Comment:
These two variables *must* be specified to be of type `string`.

Reply:
I think I beat you to it:  a679065